genesyscloud/tfexporter/genesyscloud_resource_exporter.go

```diff
@@ -316,269 +319,15 @@ func (g *GenesysCloudResourceExporter) generateOutputFiles() diag.Diagnostics {
 	return nil
 }
 
-/*****ADDED CODE HERE*****/
```

Comment:
This code was moved to different files that linked closer to how they were being used.